shell/detail/provisioning.cattle.io.cluster.vue

```diff
@@ -517,10 +517,6 @@ export default {
         return this.extDetailTabs.registration;
       }
 
-      if ( this.value.isHostedKubernetesProvider && !this.isClusterReady ) {
```

Comment:
This line looks like it was specifically put in to show the registration at the time someone is asking for it to now show. Can you find the PR that it was introduced in?

Reply:
for change on line 520:
https://github.com/aalves08/dashboard/commit/c41223692ac74680073f6efedb1e0ea197a5933d

for change inside the if condition:
https://github.com/aalves08/dashboard/commit/c9e1a19396cc227095d251f33426341a5a095967

Reply:
That's the blame, but there's still a check to see if eks/aks. We need to work out why we were showing the registration for those host types

```
if ( ( this.value.isAKS || this.value.isEKS ) && !this.isClusterReady ) {
--->
if ( this.value.isHostedKubernetesProvider && !this.isClusterReady ) {
```

Reply:
@richard-cox , using blame I saw when @jordojordo made the latest change and I checked out a commit prior to that so I could continue the trace back. This led me to https://github.com/aalves08/dashboard/commit/8a09180184974f809fcf73eaabf69e196b99373a, which belongs to PR https://github.com/rancher/dashboard/pull/6226, which fixes 3 issues:

https://github.com/rancher/dashboard/issues/6036
https://github.com/rancher/dashboard/issues/4545
https://github.com/rancher/dashboard/issues/6112

💦 💦 💦 

This is really confusing to me. Are we supposed to show the `registration` tab when creating a cluster in a hosted Kubernetes provider or not?

Maybe @nwmac can give some help here...

Reply:
Good sleuthing!  

I think the differentiator is private hosted clusters where Rancher cannot reach out to... and the registration commands are required in order for the private hosted cluster to reach back to Rancher.

So the solution required should not hide it for that type, i'm not sure how that's detectable though. Investigation required